config/environments/development.rb

```diff
@@ -45,19 +45,6 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  # config.assets.debug = true
-
-  # Suppress logger output for asset requests.
-  # config.assets.quiet = true
-
-  # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
-
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
```

Comment:
why did you delete this?

Reply:
Because I thought that it would be necessary to add configurations, and if there are many comments, keys are poorly tracked. But then I saw that the key is already present but I forgot to return the deleted block of comments.